src/main/resources/application.properties

```diff
@@ -1,10 +1,10 @@
 ## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres_demo
-spring.datasource.username= postgres
-spring.datasource.password=
+spring.datasource.url=jdbc:postgresql://${DB_URL}:5432/${DB_USER}
```

Comment:
Looks good. Just a couple of suggestions from me:
* DB url is not considered a sensitive data and can be stored in application.properties. Same for database name
* Seems like you use the same env variable for the db username and database name which is not always the case. Please consider separating them

Reply:
I made these variables more likely for more convenient use so that when changing, for example, the username, I would change it only in 1 place.  I also plan to add a variable for the database name to .env.